packages/table/src/Row/ExpandableContent.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { getAllByRole } from '@testing-library/dom';
+import { fireEvent, render } from '@testing-library/react';
+
+import { Cell } from '../Cell';
+import ExpandedContent from '../ExpandedContent/ExpandedContent';
+import TableBody from '../TableBody';
+import { LeafyGreenTableRow } from '../useLeafyGreenTable';
+import { Person } from '../utils/makeData';
+import { useTestHookCall } from '../utils/testHookCalls';
+import Table, { flexRender } from '..';
+
+import Row from '.';
+
+const RowWithExpandableContent = () => {
+  const { containerRef, table } = useTestHookCall({
+    rowProps: {
+      // eslint-disable-next-line react/display-name
+      renderExpandedContent: (_: LeafyGreenTableRow<Person>) => {
+        return <>Expandable content test</>;
+      },
+    },
+  });
+
+  return (
+    <div ref={containerRef}>
+      <Table table={table}>
+        <thead>
+          {table.getHeaderGroups().map(headerGroup => (
+            <tr key={headerGroup.id}>
+              {headerGroup.headers.map((header, i) => {
+                return <th key={header.id}>TH {i}</th>;
+              })}
+            </tr>
+          ))}
+        </thead>
+        <TableBody>
+          {table.getRowModel().rows.map((row: LeafyGreenTableRow<Person>) => {
+            return (
+              <Row key={row.id} row={row}>
+                {row.getVisibleCells().map(cell => {
+                  return (
+                    <Cell key={cell.id}>
+                      {flexRender(
+                        cell.column.columnDef.cell,
+                        cell.getContext(),
+                      )}
+                    </Cell>
+                  );
+                })}
+                {row.original.renderExpandedContent && (
+                  <ExpandedContent row={row} />
+                )}
+              </Row>
+            );
+          })}
+        </TableBody>
+      </Table>
+    </div>
+  );
+};
+
+describe('packages/table/Row/ExpandableContent', () => {
```

Comment:
Don't love how these tests are broken out -- ExpandableContent and NestedRows are not components, so this is confusing. Let's test these functionalities in a test suite that is named after the component that it is directly testing.

Reply:
I did this since the test render functions for each of these use cases can get pretty long, so I thought breaking it out by functionality (since they're pretty clearly separate) would be better than having a 500+ line test file